classes/sota_freedom-u540.bbclass

```diff
@@ -0,0 +1,18 @@
+# Support fitImage by default
+KERNEL_CLASSES_sota = " kernel-fitimage "
+KERNEL_IMAGETYPE_sota = "fitImage"
+KERNEL_IMAGETYPES_remove_sota = "uImage"
+OSTREE_KERNEL = "${KERNEL_IMAGETYPE}-${INITRAMFS_IMAGE}-${MACHINE}-${MACHINE}"
+
+OSTREE_BOOTLOADER ?= "u-boot"
+INITRAMFS_FSTYPES = "cpio.gz"
+PREFERRED_PROVIDER_virtual/bootloader_sota ?= "u-boot"
+
+IMAGE_BOOT_FILES_sota ?= "fw_payload.bin boot.scr uEnv.txt"
+KERNEL_DEVICETREE_sota ?= "sifive/${RISCV_SBI_FDT}"
+
+UBOOT_DTB_LOADADDRESS = "0x82200000"
```

Comment:
This line shouldn't be required now either

Reply:
Yup, will refresh, thanks!